doc/using/using.rst

```diff
@@ -12,48 +12,6 @@ own repository.
 For a more complete list of sample repositories for use with Binder, see the
 `Sample Binder Repositories <sample_repos.html>`_ page.
 
-Choose from many open source languages
```

Comment:
Where did this information end up in the re-org?

Reply:
Both of those topics are now covered in a how-to - good point that we don't want the top-level information to go away though. I've added a sentence linking to those guides in the landing page and on the "getting started" tutorial.